smarts/env/gymnasium/platoon_env.py

```diff
@@ -33,14 +33,20 @@
 )
 from smarts.core.controllers import ActionSpaceType
 from smarts.env.gymnasium.hiway_env_v1 import HiWayEnvV1, SumoOptions
+from smarts.env.gymnasium.wrappers.limit_relative_target_pose import (
+    LimitRelativeTargetPose,
+)
```

Comment:
It looks like this is missing a file `limit_relative_target_pose.py`

```bash
smarts/env/gymnasium/platoon_env.py:36: in <module>
    from smarts.env.gymnasium.wrappers.limit_relative_target_pose import (
E   ModuleNotFoundError: No module named 'smarts.env.gymnasium.wrappers.limit_relative_target_pose'
```

Reply:
In order to make the pull request more modular, I had separated PR #1973 (handles rl example and env upgrades for Driving SMARTS 2023.1 and Driving SMARTS 2023.2) and PR #1974 (handles rl example and env upgrades for Driving SMARTS 2023.3). The file `limit_relative_target_pose.py` was actually newly added in PR #1973. It should be okay when we (i) first merge #1973 into master, then (ii) update PR #1974 before merging it into master. In hindsight, I should have changed the base branch of this PR to point at PR #1973.